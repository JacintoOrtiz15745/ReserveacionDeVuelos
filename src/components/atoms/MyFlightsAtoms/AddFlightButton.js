import React from 'react';
import {TouchableOpacity} from 'react-native';
import PlusIcon from 'react-native-vector-icons/Entypo';
import {styles} from '../../../styles/MyFlightsStyles';
import {colors} from '../../../styles/Colors';
import {BookingConstants} from '../../../utils/Constants';
const AddFlightButton = ({navigation}) => {
  return (
    <TouchableOpacity
      style={styles.circleButton}
      testID="view"
      onPress={() => navigation.navigate(BookingConstants.BookingNowRoute)}>
      <PlusIcon name="plus" color={colors.white} size={50} />
    </TouchableOpacity>
  );
};

export default AddFlightButton;
