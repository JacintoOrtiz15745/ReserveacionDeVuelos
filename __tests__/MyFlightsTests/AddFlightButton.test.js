import React from 'react'
import {shallow} from 'enzyme'
import AddFlightButton from '../../src/components/atoms/MyFlightsAtoms/AddFlightButton'

describe('Render correctly', () => {
    const wrapper = shallow(<AddFlightButton/>)
    test('snapshot', () => {
        expect(wrapper).toMatchSnapshot();
      });
    test('Render correctly', () =>{
        const render = wrapper.findWhere((node) => node.prop('testID') === 'view');
        expect(render.exists()).toBeTruthy()
    })

})

