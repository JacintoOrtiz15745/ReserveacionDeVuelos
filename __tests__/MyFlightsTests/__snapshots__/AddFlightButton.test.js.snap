// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly snapshot 1`] = `
<ForwardRef
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "center",
      "backgroundColor": "#6472ee",
      "borderRadius": 30,
      "flexDirection": "row",
      "height": 60,
      "justifyContent": "center",
      "width": 60,
    }
  }
  testID="view"
>
  <Icon
    allowFontScaling={false}
    color="#FFF"
    name="plus"
    size={50}
  />
</ForwardRef>
`;
